package/CHANGELOG

```diff
@@ -14,11 +14,16 @@ The rules for this file:
 
 ------------------------------------------------------------------------------
 
-??/??/?? IAlibay, jaclark5, MohitKumar020291, orionarcher, xhgchen, ztimol
+??/??/?? IAlibay, jaclark5, MohitKumar020291, orionarcher, xhgchen,
+         ztimol, orbeckst
+
 
  * 2.6.0
 
 Fixes
+  * Fix AtomGroup.center_of_charge(..., unwrap=True) giving wrong
+    results on Intel macOS by removing `-ffast-math` compiler flag
+    (Issues #3821, #4211)    
```

Comment:
Not sure how we deal with this stuff, but is it worth adding an entry under "Changed" too?

Reply:
Yeah, I wasn't sure, especially now considering that @hmacdope said in https://github.com/MDAnalysis/mdanalysis/issues/4211#issuecomment-1659757345 that the failing test is not scientifically wrong (under PBC), just technically wrong.

Reply:
I updated CHANGELOG and also included @hmacdope suggestion to include the URL of the original blog post. That's a bit unusual for a CHANGELOG but in this case it's so esoteric that it seems justified and easier than having to comb through the linked issue.